MailCore/Cache/ContactManager/ContactManager.swift

```diff
@@ -85,6 +86,7 @@ public final class ContactManager: ObservableObject, ContactManageable {
             ]
         )
         backgroundRealm = BackgroundRealm(configuration: realmConfiguration)
+        transactionExecutor = TransactionExecutor(realmAccessible: backgroundRealm)
```

Comment:
For now the `backgroundRealm` is the only object that can return a realm, and this should be split up. But it should be done in a further PR IMO.